src/executables/util_cs.c

```diff
@@ -3192,13 +3198,35 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
   if (master_node_name != NULL)
     {
       check_master_info = true;
+      ha_node_list_p = prm_get_string_value (PRM_ID_HA_NODE_LIST);
+      start_node = (char *) strchr (ha_node_list_p, '@');
+      if (start_node == NULL || ha_node_list_p == start_node)
+	{
+	  return NULL;
```

Comment:
It seems to need an error handling and `return EXIT_FAILURE;`

Reply:
Right. when error arise, we call `return EXIT_FAILURE;` I will change it.